js/classes/entities/foreground/loading_animation.js

```diff
@@ -0,0 +1,69 @@
+class LoadingAnimation {
+    constructor() {
+        const center = new Vector(CANVAS.width / 2, CANVAS.height / 2);
+        
+        this.circles = [
+            { x: center.x - 50, y: center.y, dy: 20},
```

Comment:
Can you explain how this works?

Reply:
Each of the circles are updated one at a time, but for that they need these individual properties